src/js/botc/roles.ts

```diff
@@ -36,7 +36,7 @@ const RoleTypes = [
 ] as const;
 export type RoleType = (typeof RoleTypes)[number];
 
-const Editions = ["tb", "snv", "bmr", "other"] as const;
+export const Editions = ["tb", "snv", "bmr", "other"] as const;
```

Comment:
> `'Editions'` is assigned a value but only used as a type. Allowed unused vars must match `/^_/u`. [@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)

Reply:
This seems fine; we could inline it but that seems uglier than having the definition available if needed, just like RoleTypes and RoleType.